src/test/java/teammates/test/cases/action/InstructorFeedbackSubmissionEditSaveActionTest.java

```diff
@@ -35,15 +35,10 @@ protected String getActionUri() {
     }
 
     @Override
-    protected void prepareTestData() {
-        dataBundle = loadDataBundle("/InstructorFeedbackSubmissionEditSaveActionTest.json");
```

Comment:
Instead of moving this, maybe you can call `super.prepareTestData()` here instead?